tests/legacy-cli/e2e/tests/commands/config/config-global-completion.ts

```diff
@@ -0,0 +1,18 @@
+import { homedir } from 'os';
+import * as path from 'path';
+import { deleteFile, expectFileToExist } from '../../../utils/fs';
+import { ng } from '../../../utils/process';
+import { expectToFail } from '../../../utils/utils';
+
+export default async function () {
+  await expectToFail(() => ng('config', 'cli.completion.prompted', 'true'));
```

Comment:
Can you please validate the failure, something like?

```suggestion
  {
    const { message } = await expectToFail(() =>
      silentNg('config', 'cli.completion.prompted', 'true'),
    );

    if (!message.includes('Data path "/cli" must NOT have additional properties(completion).')) {
      throw new Error('Should have failued with must NOT have additional properties(completion).');
    }
  }
```